src/main/python/keyboard/keyboard_to_nt.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) 2023 FRC 5990 Permission is hereby granted, free of charge, to any person obtaining a copy of this software and
+# associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit
+# persons to whom the Software is furnished to do so, subject to the following conditions: The above copyright notice and this
+# permission notice shall be included in all copies or substantial portions of the Software. THE SOFTWARE IS PROVIDED "AS IS",
+# WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+# CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+# WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+# Lets go Trigon
+
+import sys
+import keyboard
+# if this library is not installed, dont install ntcore but pyntcore
+import ntcore
+import time
+
+TEAM_NUMBER = 4590  # GREENBLITZ 🐐🐐🐐🐐
+CLIENT_NAME = "KeyboardToNetworkTables"
+CONNECTION_TIMEOUT_DURATION = 60
+ONE_SECOND_DURATION = 1
+KEYBOARD_CHECKING_COOLDOWN_SECONDS = 0.01
+KEYBOARD_TABLE = "Keyboard"
+KEYBOARD_KEYS_TABLE = "Keyboard/Keys"
+
+
+def is_pressed(event: keyboard.KeyboardEvent):
+    return event.event_type == keyboard.KEY_DOWN
+
+
+def on_action(event: keyboard.KeyboardEvent, table: ntcore.NetworkTable):
```

Comment:
```suggestion
def handel_keyboard_event(event: keyboard.KeyboardEvent, table: ntcore.NetworkTable):
```
or something like this